web/src/features/charts/elements/GraphBackground.tsx

```diff
@@ -1,11 +1,22 @@
 /* eslint-disable react/jsx-handler-names */
 /* eslint-disable unicorn/no-null */
 /* eslint-disable react/display-name */
+import { ScaleLinear, ScaleTime } from 'd3-scale';
 import React from 'react';
 
 import { detectHoveredDatapointIndex, noop } from '../graphUtils';
 
-const GraphBackground = React.memo(
+interface GraphBackgroundProps {
+  timeScale: ScaleTime<number, number>;
+  valueScale: ScaleLinear<number, number>;
+  datetimes: Date[];
+  mouseMoveHandler?: (index: number | null) => void;
+  mouseOutHandler?: () => void;
+  isMobile: boolean;
+  svgNode: SVGElement;
+}
+
+const GraphBackground: React.FC<GraphBackgroundProps> = React.memo(
   ({
```

Comment:
```suggestion
const GraphBackground = React.memo(function graphBackground({
```
And then the props in the inner function as types.

Reply:
or we can just turn of this annoying ESLint rule...?